### PR TITLE
Transformations & Dropout added to the Model, Results and Plots Uploaded

### PR DESCRIPTION
**Pull request by arsalananwar11**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -23,71 +36,67 @@ def get_optimizer(model, optimizer_type, lr, momentum, weight_decay):
...
 
-    
-
 def get_scheduler(optimizer, scheduler_type):
     if scheduler_type == "ExponentialLR":
         return optim.lr_scheduler.ExponentialLR(optimizer, gamma=0.9)
     elif scheduler_type == "MultiStepLR":
         return optim.lr_scheduler.MultiStepLR(optimizer, milestones=[5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60], gamma=0.9)
-    
+    elif scheduler_type == "CosineAnnealingLR":
```
Comment (tkobil) [inline main.py:49]: mind just adding this to the readme under scheduler options?

File: main.py [hunk 2]
```diff
@@ -160,14 +183,11 @@ def main():
...
                 train_outfile.write(train_line)
                 
-                test_loss, test_acc = test(model, testloader, criterion, device)
+                test_loss, test_acc = test(model, testloader, criterion, device, epoch, args)
                 test_line = f"{epoch}, {test_loss}, {test_acc}\n"
                 test_outfile.write(test_line)
                 
                 print(f"TRAIN ACC: {train_acc}, TEST ACC: {test_acc}")
-                
-    if (args.save):
```
Comment (tkobil) [inline main.py:169]: mind just removing this arg from the readme then? and from the argparse params?

File: main.py (hunk 1 above)
Comment (arsalananwar11) [reply main.py:49]: Sure

**Review by tkobil — APPROVED** [review]
